src/main/resources/config.yml

```diff
@@ -34,3 +34,4 @@ layers:
       - !<particles>
         <<: *vomit
     sections: [ l1s1,l1s2,l1s3,l1s4,l1s5,l1s6 ]
+hubSectionName: city
```

Comment:
Not sure if this has been discussed, but it might make sense in a future change to make this more general. You could imagine that the seeker camp might be treated as a hub in the future as well as orth, or that there might be multiple hubs in a single layer.

Reply:
Hub is essentially only ever going to be Orth. We'll ever allow you to access your orth storage down in the abyss, and if we do ever have unique storage locations or hubs within the abyss they'll likely be handled through worldguard regions. Either way let's cross that bridge when we get there, we'll have a better idea of what we want from the code then.

Reply:
Yea, for sure. I wish github had a way to mark a comment as No Change Needed for musings...